helm/values-uat.yaml

```diff
@@ -99,12 +99,12 @@ microservice-chart:
     BLOB_STORAGE_ACCOUNT_ENDPOINT: 'https://pagopauweureceiptsfnsa.blob.core.windows.net'
     BLOB_STORAGE_CONTAINER_NAME: 'pagopa-u-weu-receipts-azure-blob-receipt-st-attach'
     ENABLE_ECS_CONSOLE: "true"
-    OTEL_RESOURCE_ATTRIBUTES: "service.name=pagopareceiptspdfserviceotl,deployment.environment=uat"
-    OTEL_EXPORTER_OTLP_ENDPOINT: "http://otel-collector.elastic-system.svc:4317"
-    OTEL_LOGS_EXPORTER: none
-    OTEL_TRACES_SAMPLER: "always_on"
+    QUARKUS_OTEL_EXPORTER_OTLP_ENDPOINT: "http://otel-collector.elastic-system.svc:4317"
+    QUARKUS_OTEL_TRACES_SAMPLER: "traceidratio"
+    QUARKUS_OTEL_TRACES_SAMPLER_ARG: "0.5"
```

Comment:
```suggestion
```
remove the sampling

Reply:
done as QUARKUS_OTEL_TRACES_SAMPLER_ARG: "1.0"